orgbee/src/app/login/login.component.html

```diff
@@ -39,64 +54,137 @@ <h4>
-          </div>
-          <div>
-            <label for="signup-email">Email</label>
-            <input type="email" formControlName="email" />
-          </div>
-        </div>
-
-        <div class="form-row">
-          <div>
-            <label for="signup-bday">Birthday</label>
-            <input type="date" formControlName="bday" />
-          </div>
-
-          <div>
-            <label for="signup-gender">Gender</label>
-            <select formControlName="gender">
-              <option value="" disabled selected>Select your gender</option>
-              <option value="male">Male</option>
-              <option value="female">Female</option>
-              <option value="other">Other</option>
-            </select>
+  
+          <div class="form-row">
+            <div>
+              <label for="signup-bday">Birthday</label>
+              <input type="date" formControlName="bday" />
+              @if(bdayControl.invalid && bdayControl.touched){
+                <div class="inc">
+                    @if(bdayControl.errors?.['required']){
+                        <div>Birth date is required</div>
+                    }
+                </div>
+              }
+            </div>
+  
+            <div>
+              <label for="signup-gender">Gender</label>
+              <select formControlName="gender">
+                <option value="" disabled selected>Select your gender</option>
+                <option value="male">Male</option>
+                <option value="female">Female</option>
+                <option value="other">Other</option>
+              </select>
+              @if(genderControl.invalid && genderControl.touched){
+                <div class="inc">
+                    @if(genderControl.errors?.['required']){
+                        <div>Gender is required</div>
+                    }
+                </div>
+              }
+            </div>
           </div>
-        </div>
-
-        <div class="form-row">
-          <div>
-            <label for="signup-password">Password</label>
-            <input type="password" formControlName="password" />
-          </div>
-          <div>
-            <label for="signup-confirm-password">Confirm Password</label>
-            <input type="password" formControlName="confirmPassword" />
+  
+          <div class="form-row">
+            <div>
+              <label for="signup-password">Password</label>
+              <input type="password" formControlName="password" minlength="8"/>
+              @if(signInPasswordControl.invalid && signInPasswordControl.touched){
+                <div class="inc">
+                    @if(signInPasswordControl.errors?.['required']){
+                        <div>Password is Required.</div>
+                    }
+                    @if(signInPasswordControl.errors?.['minlength']){
+                        <div>Password must be 8 characters long</div>
+                    }
```

Comment:
the same with here, signInPasswordControl in sign up form, maybe there's a typo somewhere in your typescript, and if not then change it to signUpPasswordControl